src/averageAges.js

```diff
@@ -14,12 +14,15 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) => (person.sex === 'm' && century)
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
```

Comment:
You use `person.sex === 'm'` 3 times. 
Try to simplify the logic. Filter the array of people by gender and then in the same filter check if you have the century flag and if so, add more logic related to century

For example:
```
const men = people.filter(person => (
     person.sex === 'm' 
     && (century ?  Math.ceil(person.died / 100) === century : true)
));

```

Reply:
👌 fixed